### ISSUE TITLE
Issues in Dropdown Column Renderer

### ISSUE THREAD

**Issue by Quernest** [lumada-design/hv-uikit-react#3975]
### Which version of NEXT UI Kit are you opening this bug for?

v5.x

### Latest version

- [X] I have tested the latest version

### No Hitachi confidential content

- [X] I'm not including any Hitachi confidential content

### Current behavior 😯

I identified two issues with the [Dropdown Column Renderer](https://lumada-design.github.io/uikit/master/?path=/docs/visualizations-table-table-renderers-dropdown-column-renderer--docs) in tables:

1. When there are 2-3 rows, the dropdown is incorrectly displayed, appearing within the table and getting cropped.
2. When no row data is provided for the column, the page crashes with the following error:

```text
TypeError: Cannot read properties of undefined (reading 'length')
at Cell (renderers.tsx:226:27)
at renderWithHooks (react-dom.development.js:16305:18)
```

### Expected behavior 🤔

1. The dropdown should display appropriately, even when there are 2-3 rows, without getting cropped.
2. The page should handle scenarios where no row data is provided for the column without crashing.

### Steps to reproduce 🕹

**For the Dropdown Display Issue:**
1. Create a table with 2-3 rows.
2. Add a Dropdown Column Renderer to the table.
3. Observe the display of the dropdown in the specified condition.

**Recording:**

https://github.com/lumada-design/hv-uikit-react/assets/21086792/51f20aae-1b81-44d2-97f6-925ef6695155

**For the Page Crash Issue:**
1. Ensure no row data is provided for the column.
2. Load or refresh the page.
3. Observe the error and resulting crash.

For example:
```tsx
<Table
  title="Data"
  columns={[
    {
      Header: 'Column Name',
      accessor: 'columnName',
    },
    hvDropdownColumn<any, string>(
      {
        Header: 'Input Type',
        accessor: 'inputType',
        id: 'inputType',
      },
      undefined,
      'Select Input Type...',
      'Select Input Type...',
      () => console.log('select me'),
    ),
    hvDropdownColumn<any, string>(
      {
        Header: 'Severity',
        accessor: 'severity',
        id: 'severity-header',
      },
      undefined,
      'Select severity...',
      'Select severity...',
      () => console.log('select me'),
    ),
  ]}
  data={[
    {
      columnName: 'State Code',
      // inputType: [], <- IS MISSED
      severity: getDropdownOptions(
        ['Critical', 'Major', 'Average', 'Minor'],
        getOption(['Critical', 'Major', 'Average', 'Minor'], 0),
      ),
    },
    {
      columnName: 'State Name',
      inputType: getDropdownOptions(
        ['Critical', 'Major', 'Average', 'Minor'],
        getOption(['Critical', 'Major', 'Average', 'Minor'], 0),
      ),
      severity: getDropdownOptions(
        ['Critical', 'Major', 'Average', 'Minor'],
        getOption(['Critical', 'Major', 'Average', 'Minor'], 1),
      ),
    },
  ]}
/>
```

**Comment by MEsteves22** [comment]
Hello @Quernest 👋

We'll take a look at these issues and let you know when fixes are available. Thanks!

**Comment by MEsteves22** [comment]
Hello 👋

These bugs were fixed in [@hitachivantara/uikit-react-core@5.44.8](https://github.com/lumada-design/hv-uikit-react/releases/tag/%40hitachivantara%2Fuikit-react-core%405.44.8). Thanks for opening this issue!
